src/settings/StaffSlips/StaffSlipForm.test.js

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import {
+  render,
+  screen,
+  fireEvent,
+} from '@testing-library/react';
+
+import '../../../test/jest/__mock__';
+
+import { Field } from 'react-final-form';
+
+import { TemplateEditor } from '@folio/stripes-template-editor';
+import {
+  KeyValue,
+  Pane,
+  Paneset,
+  TextArea,
+} from '@folio/stripes/components';
+
+import StaffSlipForm from './StaffSlipForm';
+import TokensList from './TokensList';
+import tokens from './tokens';
+import {
+  CancelButton,
+  FooterPane,
+} from '../components';
+
+jest.mock('@folio/stripes/final-form', () => jest.fn(() => (component) => component));
+jest.mock('../components', () => ({
+  CancelButton: jest.fn(() => null),
+  FooterPane: jest.fn(() => null),
+}));
+Field.mockImplementation(jest.fn(() => null));
```

Comment:
Yeah, in this case we need it, because if we don't do that, we will have tests fail with error `Cannot call a class as a function`. I tried to modify general mock, but in this case some other tests failed.

Reply:
I see, ok